### PR TITLE
Readds the boxstation whiteship as a ruin.

### PR DESCRIPTION
**Pull request by WJohn**
Bawhoppen wanted this relic to stay around because it's iconic. You decide if it should stick around as a ruin, or just as a shuttle manipulator template like it is right now.

Ya'll know exactly what this ship looks like already. It's the guitar white ship from box.

:cl: WJohn
add: An old cruiser class vessel has resiliently stuck around, and may do so for the foreseeable future.
/:cl:

### CODE REVIEW

File: code/datums/ruins/space.dm [hunk 1]
```diff
@@ -255,3 +255,10 @@
 	suffix = "gondolaasteroid.dmm"
 	name = "Gondoland"
 	description = "Just an ordinary rock- wait, what's that thing?"
+
+/datum/map_template/ruin/space/whiteshipruin_box
+	id = "whiteshipruin_box"
+	suffix = "whiteshipruin_box.dmm"
+	name "NT Medical Ship"
```
Comment (ShizCalev) [inline code/datums/ruins/space.dm]: name =

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/datums/ruins/space.dm (hunk 1 above)
Comment (WJohn) [reply code/datums/ruins/space.dm]: whoops

**Review by ShizCalev — APPROVED** [review]

**Comment by AnonymousNow** [comment]
I like the ship, and want more viable ships (especially if we can repair existing ship/shuttle ruins), so I'm all for this.

**Comment by WJohn** [comment]
Note that this ship in its current state cannot be flown, that'll take a bit of changing to make that happen. I'd have to reorganize a dedicated box/meta/pubby etc whiteship area for all of them as well as unique consoles specific to them. Wouldn't be hard to do, ultimately, but right now the ship is just for show.

**Comment by AnonymousNow** [comment]
Fair enough. Maybe something for the future.

**Comment by BlueNothing** [comment]
Could be kind of cool to make a fleet of White Ships and dock them with the station when multi-ship merge is a thing. Of course, if that happens, expect rounds to get a looooot longer, because if there are two or three distinct safe positions, as well as, possibly, a colonized derelict or a trade route established with, say, a golem ship or the mining station once it gets the ORM over there or a new one built, then most departments wind up not needing the main station. Cargo is at a major disadvantage, so guns will be kind of rare, but mining can mass-produce PKAs, and research can make mods for them. Mining can also make space suits, even if there's no other way to make them. So even without the supply shuttle (useful as it is), they'd be doing pretty alright. Assistants are pretty happy anywhere, heads are mostly happy where their departments are happy. QM becomes a miner, cargotechs become assistants or miners, miners mine. Service and civilian jobs are mostly unaffected, except curator.  Engineers focus on the derelict. Medical sets up everywhere, with genetics likely on the Meta whiteship or derelict, due to their relative safety. Science is pretty open to working anywhere, but toxins will be pretty limited by lacking TTVs, and xenobio might run into some trouble, except as an even more extensive renovation project than the derelict itself. RnD and Robotics work basically anywhere. AI is disadvantaged, but borgs also work just fine.

**Pull request merged**